docs/pages/eas/workflows/syntax.mdx

```diff
@@ -824,6 +845,20 @@ jobs:
         # @end #
 ```
 
+### `jobs.<job_id>.steps.<step>.shell`
+
+The shell to use for running the command. If not specified, the default shell for the operating system will be used.
```

Comment:
This isn't precisely true. On macOS the default shell is ZSH. However, we're running scripts with Bash.

https://github.com/expo/eas-build/blob/577e707dee03a81f5ff9f26627717f418717989c/packages/steps/src/utils/shell/command.ts#L3-L7

I think we can just say the default is Bash.